api/Controllers/AuthController.cs

```diff
@@ -51,14 +31,14 @@ public IActionResult Login([FromBody]Login request)
             if (user != null)
             {
                 var token = GenerateJWT(user);
-                return Ok(new {access_token = token});
+                return Ok( token );
             }
             return Unauthorized();
         }
 
         private Account AuthenticateUser(string email, string password)
         {
-            return Accounts.SingleOrDefault(u => u.Email == email && u.Password == password);
+            return _hotelsDb.Accounts.SingleOrDefault(u => u.Email == email && u.Password == password);
```

Comment:
What could go wrong if you store passwords in your database like this?

Reply:
Would like to do this part along with registration logic, 'cause now there is not even a method to register user